docs/source/tutorial/04-execute-the-query.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "4. Execute the query"
+---
+
+To execute the query, you will need to create an `ApolloClient` and make a new call with your query.
+
+## Create an ApolloClient
+
+Create a new file named Apollo.kt in the `com.example.rocketreserver` package and add the ApolloClient:
+
+```kotlin:title=app/src/main/java/com/example/rocketreserver/Apollo.kt
+val apolloClient = ApolloClient.builder()
+    .serverUrl("https://apollo-fullstack-tutorial.herokuapp.com")
+    .build()
+```
+
+`apolloClient` is a top-level variable so that it can be reused from other parts of the app later on for better performance. Reusing the `apolloClient` allows to reuse the underlying OkHttp instance and the associated threadpools and connections.
```

Comment:
For these, we could maybe annotate them via ⚠️  or ℹ️ emoji.

Reply:
I think in the iOS stuff I've been using:

`> **NOTE**: Something important please read it I beg you`

format